### PR TITLE
Various minor revisions to submission forms. Resolves the following tickets:

### PR DESCRIPTION
**Pull request by mikedaul**
- RDR-353
- RDR-355
- RDR-356
- RDR-357
- RDR-358
- RDR-362
- RDR-354

### CODE REVIEW

**Comment by mikedaul** [comment]
Thanks @seanaery - I've updated accordingly.

File: config/locales/rdr.en.yml [hunk 1]
```diff
@@ -71,58 +71,56 @@ en:
...
+        question12_label: "Please enter one or more key words to describe your study/dataset (e.g. Mechanical engineering, Sociology, Medical imaging, etc.). Please separate each key word with a semicolon."
+        question13_label: "Please list the geographic area(s) where your study data was collected if significant for the analysis of the data (country, state, locality)."
+        question14_label: "Please list the dates of data collection."
+        question15_label: "Please list any languages associated with this study/dataset.  If multiple languages, please separate each language with a semicolon."
+        question16_label: "Please list the formats associated with the materials in this deposit (e.g. PDF, CSV, DICOM, WAV, TIFF). Please separate each format with a semicolon."
+        question17_label: "Please list any related publications, datasets or other published materials that you would like to associate with the study/dataset record. Enter the citation and persistent identifier (DOI)."
+        question18_label: "please list funding agency information"
+        question19_label: "please list grant number and information"
+        question20_label1: "Yes"
```
Comment (coblej) [inline config/locales/rdr.en.yml]: All the other question prompts start with uppercase letters.  Also, for the grant number one, do you really mean "grant number and information" or should it just be "grant number information"?  These prompts seem terse compared to the other ones but maybe that's what the product owners want?

File: app/assets/javascripts/submissions.js [hunk 2]
```diff
@@ -48,17 +64,17 @@ function nextPrev(n) {
 
   currentTab = currentTab + n;
 
-  //console.log('current tab: ' + currentTab);
+  console.log('current tab: ' + currentTab);
```
Comment (coblej) [inline app/assets/javascripts/submissions.js]: Did you mean to leave this uncommented?

**Comment by mikedaul** [comment]
Thanks @coblej - I've made the changes and also updated submission.rb to remove :screening_license

**Comment by mikedaul** [comment]
Closing this for now - waiting for TBD text to be finalized.

**Pull request closed without merging**